### PR TITLE
Fix run polling command

### PR DESCRIPTION
**Pull request by israelpoli**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-16847

## Description
The RunPollingCommand script fails at runtime.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by tkatzir — APPROVED** [review]
Great!

File: Packs/CommonScripts/ReleaseNotes/1_7_65.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### RunPollingCommand
+- Fixed an issue where ***RunPollingCommand*** script fails at runtime.
```
Comment (tkatzir) [inline Packs/CommonScripts/ReleaseNotes/1_7_65.md:4]:
```suggestion
- Fixed an issue where ***RunPollingCommand*** script failed at runtime. 
```

**Comment by ShahafBenYakir** [comment]
@israelpoli Is this fixing [this](https://dfircommunity.slack.com/archives/CBEM7L3PV/p1663782502768249)?

**Comment by israelpoli** [comment]
> @israelpoli Is this fixing [this](https://dfircommunity.slack.com/archives/CBEM7L3PV/p1663782502768249)?

Yes that should fix it

File: Packs/CommonScripts/ReleaseNotes/1_7_65.md (hunk 1 above)
Comment (DeanArbel) [inline Packs/CommonScripts/ReleaseNotes/1_7_65.md]:
```suggestion
- Fixed an issue in ***RunPollingCommand*** where it passed malformed arguments, causing **GenericPolling** playbook to fail. 
```

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**